simpleflow/swf/task.py

```diff
@@ -211,7 +208,7 @@ def schedule(self, domain, task_list=None, executor=None, **kwargs):
             workflow_id=self.id,
             workflow_type=model,
             task_list=task_list or self.task_list,
-            control=json_dumps(control) if control is not None else None,
+            control=json_dumps(control) if control is not None else None,  # FIXME encode in the model?
```

Comment:
Yes, and maybe use `simpleflow.format()` so it gets translated to a jumbo field if it's larger than 32k?

Reply:
Yep; that's what I meant, sorry it wasn't clear enough